esx_service/utils/auth_api.py

```diff
@@ -347,6 +348,10 @@ def _tenant_create(name, description="", vm_list=None, privileges=None):
 
 def _tenant_update(name, new_name=None, description=None, default_datastore=None):
     """ API to update a tenant """
+    if name == auth_data_const.DEFAULT_TENANT:
+        error_info = error_code.generate_error_info(ErrorCode.TENANT_NAME_INVALID, name, VALID_TENANT_NAMES)
```

Comment:
If I understand this code path correctly, this change stops updating description, default datastore etc for the ```_DEFAULT``` tenant where as our intention is only to stop renaming ```_DEFAULT``` tenant, please correct me otherwise.